src/helpers/daily-data-providers/health-connect-steps.ts

```diff
@@ -0,0 +1,25 @@
+﻿import { add } from "date-fns";
+import { DailyDataQueryResult } from "../query-daily-data";
+import queryAllDeviceDataV2Aggregates from "../query-all-device-data-v2-aggregates";
+
+export async function stepsProvider(
+    startDate: Date,
+    endDate: Date,
+): Promise<DailyDataQueryResult> {
+    const aggregates = await queryAllDeviceDataV2Aggregates({
```

Comment:
Don't think this is right; which is why I said we should skip steps for now until we have aggregate support on the Android side and can establish a "Daily Steps" rule

Multiple devices can write steps into Health Connect so you will get overlapping intervals; e.g. you can have 20 steps written from your phone from 10:15 - 10:30 and then 100 steps written from your smartwatch from 10:00-11:00

The correct number of steps in that instance is 100, not 120 (which is what this would provide I believe); and Health Connect aggregates (like statistics on iOS) do that magic to dedupe intervals appropriately.

Reply:
@bxh max heart rate would be nice, I was thinking about adding that to the main dashbaord.  But for MVP RHR/Sleep would be the key ones to focus on; and once we support client-side aggregates we can do steps